packages/ia-components/package.json

```diff
@@ -21,6 +21,8 @@
         "@storybook/addon-knobs": "^4.1.4",
         "@storybook/react": "^4.1.4",
         "assert": "^1.4.1",
+        "babel-core": "^7.0.0-bridge.0",
+        "babel-jest": "^23.4.2",
```

Comment:
jest has been upgraded with the previous PR, let's make sure master is rebased into this branch